src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -48,6 +57,27 @@ export const SueReportLocation = ({
   const zipCodeInputRef = useRef();
   const cityInputRef = useRef();
 
+  const queryVariables = {
+    status:
+      'TICKET_STATUS_UNPROCESSED,TICKET_STATUS_OPEN,TICKET_STATUS_IN_PROCESS,TICKET_STATUS_WAIT_REQUESTOR,TICKET_STATUS_WAIT_THIRDPARTY'
+  };
+
+  const { data, isLoading } = useQuery([QUERY_TYPES.SUE.REQUESTS, queryVariables], () =>
+    getQuery(QUERY_TYPES.SUE.REQUESTS)(queryVariables)
+  );
```

Comment:
how much does it affect performance getting all the data for markers on the screen?

Reply:
That's hard to answer. The more data, the longer it will take. But since we add filtering according to certain statuses, we only get the filtered data from the API instead of the whole data. But of course there will be a time difference between 10 data and 1000 data. 🤷‍♂️

Reply:
So it is crucial to rely on react-queries cache here requesting the data. The cache time should have a certain time, not just five minutes, in order to not go through long request durations and waiting times in order to fill out the form.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1059/commits/6b58b969cec05a8eaa751b50e57bd9f033648eae